docs/getting-started.md

```diff
@@ -89,17 +97,17 @@ For the purposes of this demo, we will grant the `default` `ServiceAccount` admi
 kubectl create rolebinding default-admin --clusterrole=admin --serviceaccount=default:default
 ```
 
-**Note that this will grant admin privilages to the `default` `ServiceAccount` in the namespace that the command is run from, so you will only be able to
+**Note that this will grant admin privileges to the `default` `ServiceAccount` in the namespace that the command is run from, so you will only be able to
 run Workflows in the namespace where the `RoleBinding` was made.**
 
 ## 4. Run Sample Workflows
 ```sh
-argo submit --watch https://raw.githubusercontent.com/argoproj/argo/master/examples/hello-world.yaml
-argo submit --watch https://raw.githubusercontent.com/argoproj/argo/master/examples/coinflip.yaml
-argo submit --watch https://raw.githubusercontent.com/argoproj/argo/master/examples/loops-maps.yaml
-argo list
-argo get xxx-workflow-name-xxx
-argo logs xxx-pod-name-xxx #from get command above
+argo submit -n argo --watch https://raw.githubusercontent.com/argoproj/argo/master/examples/hello-world.yaml
```

Comment:
It is not necessary to submit workflows under the `argo` namespace, and in some circumstances it might not be desirable. Can we remove this? Or -- if there's a reason you added this that I'm not seeing please let me know!

Reply:
Ah, interesting. I ran into trouble submitting hello-world in the `default` namespace earlier, so I assumed they had to be in `argo`

I'll try to reproduce it tomorrow and submit a bug, along with fixing the three instances here. Thanks for the review!